.eslintrc

```diff
@@ -1,5 +1,9 @@
+
 {
     "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "warnOnUnsupportedTypeScriptVersion": false
```

Comment:
We use version `5.1.3` when latest supported is `5.1.0`. It will be supported soon but we're safe anyway